### DISCUSSION TITLE
nFeature is very high in 10x run data

### DISCUSSION THREAD

**Discussion by bl24** [satijalab/seurat#5085] [Q&A] [answered]
Hello. I'm new to scRNA and I've been looking at the guided PBMC tutorial and using that as a guide on doing analysis for data I've generated from 10x. I had a question regarding the part of the guide where the data is filtered based on QC metrics: 

pbmc <- subset(pbmc, subset = nFeature_RNA > 200 & nFeature_RNA < 2500 & percent.mt < 5)

If I try to apply "nFeature_RNA < 2500", that eliminates almost all of my cells. I was wondering if it's normal for 10x data to have such high nFeature values?

**Comment by saketkc** [comment]
Do you have a distribution plot for all the 3 features that you can share?

**Comment by bl24** [comment]
![VlnPlot](https://user-images.githubusercontent.com/14930703/133830817-e85829f2-5a73-402c-91f2-9906f404e75b.png)

Here's my violin plot for the 3 features.

**Reply by saketkc** [reply]
Looks fine. The idea of filtration in the tutorial is to remove outliers, so the cutoffs are data dependent. You might just use the mt.percent cutoff here.

**Comment by bl24** [comment, marked as answer]
Got it. Thanks!
